lib/collections/transform/cartOrder.js

```diff
@@ -166,7 +184,8 @@ export const cartOrderTransform = {
     const taxes = parseFloat(this.getTaxTotal());
     const discount = parseFloat(this.getDiscounts());
     const discountTotal = Math.max(0, subTotal - discount);
-    const total = discountTotal + shipping + taxes;
+    const surcharges = parseFloat(this.getSurcharges());
+    const total = discountTotal + shipping + taxes + surcharges;
     return accounting.toFixed(total, 2);
   },
```

Comment:
As far as I can see, `getTotal` is never called anymore, so it could be deleted and `getSurcharges` isn't needed.

Reply:
`getTaxTotal()` is also unused outside of `getTotal`, so I've removed that as well.